README.md

```diff
@@ -1 +1,126 @@
-# to-do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>To Do List</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To Do List] <a name="about-project"></a>
+
+[To Do List] is project about adding and removing your tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- **[html]**
+- **[javascript]**
+- **[css]**
+- \*\*[webpack]
+
+<p align="right">(<a href="#readme-top">back  to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+> Open GitHub and go to my GitHub repository name: to-do-list .to clone.
+> Click “Code” and copy the given URL.
+> click code and copy the URL.
+
+> click code and copy the URL.
+
+> Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+
+> Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+> Syntax:
+
+$ git clone https://github.com/memicelis/to-do-list.git
+
+### Usage
+
+To run the project, execute the following command:
+
+> open your directory to-do-list
+> to run the file :
+> \use vs-code or any IDE to view line code .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Since some unexperienced developers might be going through this project and when they want to start the project you haven't provided the necessary steps for them to do, when I try to do the steps that you're suggesting nothing happens, so please provide a detailed setup and usage of this project. For example, I cloned the project but there are not required steps on how to see the project!